basis/node/_methods.py

```diff
@@ -11,8 +11,22 @@
 
 
 @dataclass
-class Table:
-    name: str
+class TableVersion:
```

Comment:
This doesn't need to be a `dataclass` anymore, since it doesn't define any fields.